balancer/weightedroundrobin/balancer.go

```diff
@@ -339,16 +393,21 @@ type picker struct {
 	v         uint32             // incrementing value used by the scheduler; accessed atomically
 	cfg       *lbConfig          // active config when picker created
 	subConns  []*weightedSubConn // all READY subconns
+
+	// These fields are built out at creation time and read only after, so no
+	// need for synchronization.
```

Comment:
`// The following fields are immutable`?

Reply:
Sure this is how I see it documented throughout the codebase in other places but it's equivalent to yours so switched for all three structs I added this comment for.